nixos/tests/networking-proxy.nix

```diff
@@ -66,46 +66,46 @@ in import ./make-test.nix ({ pkgs, ...} : {
 
   testScript =
     ''
-      startAll;
+      start_all()
 
       # no proxy at all
-      print $machine->execute("env | grep -i proxy");
-      print $machine->execute("su - alice -c 'env | grep -i proxy'");
-      $machine->mustFail("env | grep -i proxy");
-      $machine->mustFail("su - alice -c 'env | grep -i proxy'");
+      print(machine.execute("env | grep -i proxy"))
+      print(machine.execute("su - alice -c 'env | grep -i proxy'"))
+      machine.fail("env | grep -i proxy")
+      machine.fail("su - alice -c 'env | grep -i proxy'")
 
       # Use a default proxy option
-      print $machine2->execute("env | grep -i proxy");
-      print $machine2->execute("su - alice -c 'env | grep -i proxy'");
-      $machine2->mustSucceed("env | grep -i proxy");
-      $machine2->mustSucceed("su - alice -c 'env | grep -i proxy'");
+      print(machine2.execute("env | grep -i proxy"))
+      print(machine2.execute("su - alice -c 'env | grep -i proxy'"))
+      machine2.succeed("env | grep -i proxy")
+      machine2.succeed("su - alice -c 'env | grep -i proxy'")
 
       # explicitly set each proxy option
-      print $machine3->execute("env | grep -i proxy");
-      print $machine3->execute("su - alice -c 'env | grep -i proxy'");
-      $machine3->mustSucceed("env | grep -i http_proxy | grep 123");
-      $machine3->mustSucceed("env | grep -i https_proxy | grep 456");
-      $machine3->mustSucceed("env | grep -i rsync_proxy | grep 789");
-      $machine3->mustSucceed("env | grep -i ftp_proxy | grep 101112");
-      $machine3->mustSucceed("env | grep -i no_proxy | grep 131415");
-      $machine3->mustSucceed("su - alice -c 'env | grep -i http_proxy | grep 123'");
-      $machine3->mustSucceed("su - alice -c 'env | grep -i https_proxy | grep 456'");
-      $machine3->mustSucceed("su - alice -c 'env | grep -i rsync_proxy | grep 789'");
-      $machine3->mustSucceed("su - alice -c 'env | grep -i ftp_proxy | grep 101112'");
-      $machine3->mustSucceed("su - alice -c 'env | grep -i no_proxy | grep 131415'");
+      print(machine3.execute("env | grep -i proxy"))
+      print(machine3.execute("su - alice -c 'env | grep -i proxy'"))
+      machine3.succeed("env | grep -i http_proxy | grep 123")
+      machine3.succeed("env | grep -i https_proxy | grep 456")
+      machine3.succeed("env | grep -i rsync_proxy | grep 789")
+      machine3.succeed("env | grep -i ftp_proxy | grep 101112")
+      machine3.succeed("env | grep -i no_proxy | grep 131415")
+      machine3.succeed("su - alice -c 'env | grep -i http_proxy | grep 123'")
+      machine3.succeed("su - alice -c 'env | grep -i https_proxy | grep 456'")
+      machine3.succeed("su - alice -c 'env | grep -i rsync_proxy | grep 789'")
+      machine3.succeed("su - alice -c 'env | grep -i ftp_proxy | grep 101112'")
+      machine3.succeed("su - alice -c 'env | grep -i no_proxy | grep 131415'")
 
       # set default proxy option + some other specifics
```

Comment:
instead of making this a comment, can you make this a subtest?

```python
with subtest("this title explains what you assume to happen in this block"):
    print(machine4.execute("env | grep -i proxy"))
    print(machine4.execute("su - alice -c 'env | grep -i proxy'"))
    machine4.succeed("env | grep -i http_proxy | grep 000")
    ...
```

Reply:
Done!